docs/reference/ilm/ilm-index-lifecycle.asciidoc

```diff
@@ -0,0 +1,76 @@
+[role="xpack"]
+[testenv="basic"]
+[[ilm-index-lifecycle]]
+=== Index lifecycle
+++++
+<titleabbrev>Index lifecycle</titleabbrev>
+++++
+
+{ilm-init} splits the index lifecycle into four _phases_: 
+
+* Hot--the index is actively being updated and queried.
+* Warm--the index is no longer being updated, but is still being queried.
+* Cold--the index is no longer being updated and is seldom queried. The
+information still needs to be searchable, but it's okay if those queries are
+slower.
+* Delete--the index is no longer needed and can safely be removed.
```

Comment:
Is there a reason we use em dashes here? I think a colon (`:`) might look better. It'll also let you capitalize the sentences.